CHANGES_NEXT_RELEASE

```diff
@@ -7,3 +7,4 @@
 - Fix: Service-Path supported for forwarding of registrations between Config Manager and Context Broker (Issue #719)
 - Fix: Service-Path supported for forwarding to Context Providers (Issue #719)
 - Fix: Hash sign (#) in service-path, or multiple service-paths not valid for updateContext requests (Issue #746)
+- Fix: Not forwarding the default Service-Path to context providers (Issue #714)
```

Comment:
Nope, it works, but it is *not* covered in tests, also not that a query/update without X-Auth token doesn't send an X-Auth token in the forward.